Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -2040,5 +2040,5 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
     <string name="fab_scanner_tip_title">QR-Code-Scanner</string>
     <!-- XTXT: UQS Tooltip subtitle -->
     <string name="fab_scanner_tip_subtitle">Scannen Sie Tests, Zertifikate und Check-ins mit diesem Scanner.</string>
-    <string name="home_eol_message">Der Betrieb der Corona-Warn-App wurde eingestellt. Vielen Dank für Ihre Nutzung. Mit Ihrer Unterstützung konnten während der Pandemie mehr als xx Millionen Warnungen verschickt werden. Sie können weiterhin auf Ihre bereits vorhandenen Zertifikate und das Kontakt-Tagebuch zugreifen. Alle anderen Funktionen stehen nicht mehr zur Verfügung.</string>
+    <string name="home_eol_message">Vielen Dank für die Nutzung der Corona-Warn-App. Während der Pandemie wurden durch die Nutzenden der Corona-Warn-App knapp 9 Millionen positive Testergebnisse geteilt. Durch diese Teilung positiver Testergebnisse konnten Personen schnell vor einem erhöhten Infektionsrisiko gewarnt werden, das sonst unerkannt geblieben wäre.  Viele Gewarnte wurden anschließend positiv getestet. Die Corona-Warn-App hat dank Ihrer gemeinsamen Mithilfe ihren Zweck der frühzeitigen Warnung erfüllt und die Beendigung von Infektionsketten unterstützt.\n\nSeit dem 01. Juni 2023 wird die Corona-Warn-App nicht mehr weiterentwickelt. Sie können weiterhin auf Ihre bereits vorhandenen Zertifikate und das Kontakt-Tagebuch zugreifen. Alle anderen Funktionen stehen nicht mehr zur Verfügung.</string>
```

Comment:
I prefer .... Ab dem 1. Juni 2023 ... instead of ... Seit dem 1. Juni 2023 ... => but if there is no official change leave the text as it was handed over